trypurescript.cabal

```diff
@@ -13,23 +13,23 @@ author: Phil Freeman
 data-dir: ""
 
 executable trypurescript
-    build-depends: base ==4.*,
-                   aeson -any,
-                   bytestring >=0.10.0.2 && <0.11,
-                   data-default -any,
-                   directory -any,
-                   filepath -any,
-                   Glob -any,
-                   scotty -any,
+    build-depends: base ==4.14.1.0,
```

Comment:
Can we leave the bounds as `-any` here, please? All of these dependencies are already sufficiently nailed down by `purescript` itself. The only one where I think we should keep the bounds is `scotty`.

Reply:
Meaning, if a dependency version was listed as `-any`, keep it as that? Since your comment here was made on the `base` line, I'm getting conflicting information in this comment.

Reply:
Yeah sorry, I think `base` can be left alone too.

Reply:
I've force-pushed some changes to reset earlier work. What needs to change now?